### PR TITLE
adding method to convert heading pitch roll in quaternion defined in …

### PR DESCRIPTION
**Pull request by kaktus40**
…aircraft reference axis (North East Down)

### CODE REVIEW

File: Source/Core/Transforms.js [hunk 1]
```diff
@@ -381,6 +381,36 @@ define([
...
+     * @example
+     * // Get the transform from local heading-pitch-roll at cartographic (0.0, 0.0) to Earth's fixed frame.
+     * var center = Cesium.Cartesian3.fromDegrees(0.0, 0.0);
+     * var heading = -Cesium.Math.PI_OVER_TWO;
+     * var pitch = Cesium.Math.PI_OVER_FOUR;
+     * var roll = 0.0;
+     * var transform = Cesium.Transforms.aircraftHeadingPitchRollToFixedFrame(center, heading, pitch, roll);
+     */
+    Transforms.aircraftHeadingPitchRollToFixedFrame = function(origin, heading, pitch, roll, ellipsoid, result) {
+        // checks for required parameters happen in the called functions
```
Comment (pjcozzi) [inline Source/Core/Transforms.js]: Can you still add unit tests for these with `toThrowDeveloperError`?

File: Source/Core/Transforms.js [hunk 2]
```diff
@@ -413,6 +443,35 @@ define([
...
+     * @example
+     * // Get the quaternion from local heading-pitch-roll at cartographic (0.0, 0.0) to Earth's fixed frame.
+     * var center = Cesium.Cartesian3.fromDegrees(0.0, 0.0);
+     * var heading = -Cesium.Math.PI_OVER_TWO;
+     * var pitch = Cesium.Math.PI_OVER_FOUR;
+     * var roll = 0.0;
+     * var quaternion = Cesium.Transforms.aircraftHeadingPitchRollQuaternion(center, heading, pitch, roll);
+     */
+    Transforms.aircraftHeadingPitchRollQuaternion = function(origin, heading, pitch, roll, ellipsoid, result) {
+        // checks for required parameters happen in the called functions
```
Comment (pjcozzi) [inline Source/Core/Transforms.js]: Same comment.

**Comment by pjcozzi** [comment]
Can you update [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) so users know about these new functions?

**Comment by pjcozzi** [comment]
@bagnell can you take a quick look at the math?

**Comment by pjcozzi** [comment]
Thanks again for the pull request, @kaktus40!

**Comment by kaktus40** [comment]
I just updated the files following your recommendations. It seems there is some conflicts.

**Comment by bagnell** [comment]
@pjcozzi Looks good to me.

**Comment by pjcozzi** [comment]
Thanks for the review, @bagnell!

Thanks again for the contribution, @kaktus40!

**Pull request merged**